examples/performance/00_overview.ipynb

```diff
@@ -0,0 +1,753 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Performance optimization overview\n",
+    "\n",
+    "This tutorial illustrates the performance optimizations applied to the code generated by an `Operator`. As we shall see, most optimizations are automatically applied as they're known to systematically improve performance. Others, whose impact varies across different `Operator`s, are instead to be enabled through specific options.\n",
```

Comment:
The tutorial seems to be 'explaining the optimizations flags that can be passed to `Operator`' more then 'illustrating the performance optimizations applied'?

Also, `Operator`s -> `Operator`'s

Reply:
>The tutorial seems to be 'explaining the optimizations flags that can be passed to Operator' more then 'illustrating the performance optimizations applied'?

perhaps both?

Reply:
> Also, Operators -> Operator's

fixing

Reply:
> The tutorial seems to be 'explaining the optimizations flags that can be passed to Operator' more then 'illustrating the performance optimizations applied'?

addressed in the commit I'm about to push, thanks